tests/test_agent_trainer.py

```diff
@@ -0,0 +1,45 @@
+# import gym  # noqa: F401
+import pytest
+from stable_baselines3 import PPO
+
+from imitation.policies import trainer
+from imitation.rewards import common as rewards_common
+
+
+@pytest.fixture
+def agent():
+    return PPO("MlpPolicy", env="CartPole-v1")
+
+
+@pytest.fixture
+def reward_fn() -> rewards_common.RewardFn:
```

Comment:
Do you intend to change this in the future? If not, then I don't think it needs to be a fixture, you can just have:

```
def reward_fn(obs, acts, next_obs, dones):
    return dones.astype(float)
```